"""
1. Напишите параметрический декоратор, который вызывает оборачиваемую функцию
несколько раз (по умолчанию - 2).

Предусмотрите возможность продолжать повторные вызовы декорируемой функции
при возникновении в ней исключения, в таких случаях выводите сообщения об
ошибке для каждого вызова. Используйте для этого именованный аргумент
continue_on_errors со значением по умолчанию False.


2. Напишите параметрический декоратор, который проверяет количество и типы
позиционных аргументов оборачиваемой функции.

В случае любого несоответствия необходимо сгенерировать исключение TypeError.
"""


def type_check():
    pass


def repeat():
    pass
